src/sentry/snuba/metrics/query.py

```diff
@@ -1,6 +1,6 @@
 """ Classes needed to build a metrics query. Inspired by snuba_sdk.query. """
 import math
-from collections import Mapping
+from collections.abc import Mapping
```

Comment:
i think you have outdated base branch version, this change already in master https://github.com/getsentry/sentry/pull/36195

Reply:
Thank you, yes, I accidentally merged master into this branch without updating the base branch.